tsconfig.prod.json

```diff
@@ -1,23 +1,24 @@
 {
+  "extends": "@tsconfig/node10/tsconfig.json",
+
   "compilerOptions": {
-    "target": "es2017",
-    "module": "commonjs",
     "allowJs": true,
     "declaration": true,
     "declarationMap": true,
     "resolveJsonModule": true,
     "removeComments": true,
-    "strict": true,
-    "esModuleInterop": true,
-    "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true
+    "lib": ["dom"],
```

Comment:
do we really need to have this one? We may miss some errors while trying to interact with some DOM specific API in NodeJS context

Reply:
Yes, we need it where we use URL due to a TypeScript bug. People said this would be a workaround until the TS team has fixed the issue.